portfolio/src/main/webapp/index.html

```diff
@@ -1,27 +1,183 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Hadley Shapland</title>
-    <link rel="stylesheet" href="style.css">
-    <!-- include jquery -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="script.js"></script>
-  </head>
-
-  <body>
-    <div id="sidebar"></div>
- 
-    <div id="content">
-      <h1>HADLEY SHAPLAND</h1>
-      <p>Welcome to my webpage! Check out my <a href="https://www.linkedin.com/in/hadley-shapland/">
-          LinkedIn</a> and my <a href="https://github.com/hadleyshapland">
-          GitHub</a> to learn more about me.
-      </p>
-      <h3>Contact</h3>
-      <p>hadley.a.shapland@vanderbilt.edu</p>
-      <p>217.418.9903</p>
-      
-    </div>
-  </body>
+<html lang="en">
+   <head>
+      <meta charset="utf-8" />
```

Comment:
maybe omit the closing / from the meta and link tags

Reply:
Done